base/setup/usetup/bootsup.c

```diff
@@ -2378,9 +2405,11 @@ InstallFatBootcodeToPartition(
         /* Update 'boot.ini' */
         CombinePaths(DstPath, ARRAYSIZE(DstPath), 2, SystemRootPath->Buffer, L"boot.ini");
 
+        GetBootSectFilePath(szBootSectFile, ARRAYSIZE(szBootSectFile));
+
         DPRINT1("Update 'boot.ini': %S\n", DstPath);
         Status = UpdateBootIni(DstPath,
-                               L"C:\\bootsect.ros",
+                               szBootSectFile,
```

Comment:
No, this is wrong. The correct letter is really "C:", this is what is required by NTLDR when booting a bootsector, because "C:" is also the name of the boot drive, whatever it can be.

Reply:
OK. Reverted.

Reply:
Add the explanation as a comment?

Reply:
"The boot drive is always C: drive."

Alright?

Reply:
The comment is:
"Windows' NTLDR loads an external bootsector file when the specified drive letter is C:, otherwise it will interpret it as a boot DOS path specifier."

Exercise for the reader: test this assertion on a real Windows NT/2k/2k3 installation to be convinced about that.

Reply:
OK.